Tasks/RunLoadTest/task.json

```diff
@@ -59,7 +59,18 @@
       "label": "Number of permissible threshold violations",
       "required": false,
       "helpMarkDown": "Number of threshold violations above which the load test outcome is considered unsuccessful."
-    }
+    },
+    {
+      "name": "MachineType",
+      "type": "radio",
+      "label": "Run load test using",
+      "required": true,
+      "defaultValue": "0",
+      "options": {
+        "2": "Self-provisioned agents",
```

Comment:
No.

---
In reply to: [73512668](https://github.com/Microsoft/vsts-tasks/pull/2158#discussion_r73512668) [](ancestors = 73512668)